packages/remix-server-runtime/server.ts

```diff
@@ -1010,7 +1018,23 @@ async function assertResponsesMatch(_a: Response, _b: Response) {
     assert(
       a,
       b,
-      (r) => r.text(),
+      async (r) => {
+        let text = await r.text();
+
+        // Strip stack trace from default error boundary HTML
+        if (
+          text.includes(">Application Error</h1>") &&
+          text.includes("💿 Hey developer👋")
+        ) {
+          return stripStackTraceFromDefaultErrorBoundary(text);
+        }
+
+        if (text.includes("<script>window.__remixContext")) {
+          return stripStackTraceFromRemixContext(text);
+        }
```

Comment:
Short of the above fix stack traces were the rest of the failures, so stripping them out of both `window.__remixConext` and the `<pre>` before comparing